framework/structures.md

```diff
@@ -414,7 +414,8 @@ This value MUST be a version 4 [UUID](https://www.ietf.org/rfc/rfc4122.txt).
 
 ##### Encryption Context
 
-The [custom encryption context](#encryption-context) associated with this branch key.
+The [encryption context](./structures.md#encryption-context) is a key-value pair provided by users and is associated with the branch key.
+The key of this encryption context is alway prepended with `aws-crypto-ec:`.
```

Comment:
Are we ok with this? This is very technically a breaking change right?

Reply:
I need to remove this because technically it's not true. Will commit Tony's suggestion.